atomic_reactor/inner.py

```diff
@@ -325,14 +330,28 @@ def build_docker_image(self):
                 self.autorebuild_canceled = True
                 raise
 
+            start_time = datetime.datetime.now()
+            self.plugins_timestamps['dockerbuild'] = start_time.isoformat()
```

Comment:
Why is there no way to know the TZ - is it some docker-specific issue? The other instance is using `.utcnow()` and I was assuming the result is correct.